telosnetwork/teloscan
# #176 | Fix persistent contract information when changing /address routes

# Fixes #176 

## Description
Presently, navigating from a contract's /address route to a non-contract address, the contract info persists incorrectly.

## Test scenarios
- run `yarn dev`
- go to a contract's address route, eg. http://localhost:8080/address/0x261cf410d0a83193d647e47c35178288d99e12dd
    - note the "created at ... by address ..." info in the header
- click a link to a non-contract address, eg. any row in the TX table "from" column which does not have the contract icon next to it
    - the "created at ... by address ..." info should be gone from the header
